definitions/linux.yaml

```diff
@@ -128,8 +128,8 @@ sources:
 - type: ARTIFACT
   attributes:
     names:
-      - UsersShellConfigs
-      - RootUserShellConfigs
+      - UsersShellHistory
+      - RootUserShellHistory
 labels: [Configuration Files]
```

Comment:
This is fine for this PR, I've created: https://github.com/ForensicArtifacts/artifacts/issues/123

Can you sync with HEAD and I'll merge.